src/adapters/config.ts

```diff
@@ -923,6 +923,33 @@ class Config extends Fetcher {
     )
   }
 
+  public async setJoinLeaveChannel(guildId: string, channelId: string) {
+    return await this.jsonFetch(`${API_BASE_URL}/configs/join-leave`, {
+      method: "POST",
+      body: { guildId, channelId },
+    })
+  }
+
+  public async removeJoinLeaveChannel(guildId: string) {
+    return await this.jsonFetch(`${API_BASE_URL}/configs/join-leave`, {
+      method: "DELETE",
+      body: {
+        guildId,
+      },
+    })
+  }
+
+  public async getJoinLeaveChannel(guildId: string) {
+    return await this.jsonFetch<ResponseGetVoteChannelConfigResponse>(
```

Comment:
VoteChannel for JoinLeaveChannel, is it a mistake?

Reply:
they are the same so i reused it